### DISCUSSION TITLE
Regarding delta function

### DISCUSSION THREAD

**Discussion by ryankong123** [chulminy/CIVE497-CIVE700#123] [General]
If we obtain a Fourier transform equal to 1/2[delta(f-p0)+delta(f+p0)], on a Y(f) vs. f plot, what value would the graph have at p0? would it be 1/2 or infinity? I think I confused myself because I'm looking at slide 8 of the signal processing II lecture notes and it says delta(0) = infinity, so would that mean that at p0 the point would go to infinity?

Please advise.

Thanks!

**Comment by chulminy** [comment]
Just plot the stem plot with an amplitude having 1. 

It's not critical.
